docs/proposals/ldap-group-sync.md

```diff
@@ -0,0 +1,92 @@
+# Allowing Groups to be Based on and Synced With External LDAP Records
+
+Linking internal `Group`s of users to their representations on LDAP servers allows for administrators to have an easier way to manage groups inside OpenShift. A command like `oadm sync groups` or `oadm sync group <name>` will trigger a sync of group members with their external support (if any).
+
+## Supporting Group Sources
+In order to link `Group` structures to external providers of their data, each `Group` will need an (optional) `GroupSource`:
+
+```go
+type Group struct{
+  ...
+  // Source is an optional parameter that enables the Group member list to be linked to an entry in another system or database
+  Source GroupSource
```

Comment:
Add detail here.  What sort of format is going to be required?  I assume that this would allow you to know which Groups are managed via LDAP syncing.  Does it also need to say exactly which group backs it?